samples/sample_multi_transcode/src/pipeline_transcode.cpp

```diff
@@ -2609,47 +2501,44 @@ MFX_IOPATTERN_IN_VIDEO_MEMORY : MFX_IOPATTERN_IN_SYSTEM_MEMORY);
         || (pInParams->nExtBRC && (pInParams->EncodeId == MFX_CODEC_HEVC || pInParams->EncodeId == MFX_CODEC_AVC)) ||
         pInParams->IntRefType || pInParams->IntRefCycleSize || pInParams->IntRefQPDelta || pInParams->nMaxFrameSize)
     {
-        m_CodingOption2.LookAheadDepth = pInParams->nLADepth;
-        m_CodingOption2.MaxSliceSize = pInParams->nMaxSliceSize;
-        m_CodingOption2.MaxFrameSize = pInParams->nMaxFrameSize;
-        m_CodingOption2.BRefType = pInParams->nBRefType;
+        auto co2 = m_mfxEncParams.AddExtBuffer<mfxExtCodingOption2>();
+        co2->LookAheadDepth = pInParams->nLADepth;
+        co2->MaxSliceSize = pInParams->nMaxSliceSize;
+        co2->MaxFrameSize = pInParams->nMaxFrameSize;
+        co2->BRefType = pInParams->nBRefType;
 
-        m_CodingOption2.IntRefType = pInParams->IntRefType;
-        m_CodingOption2.IntRefCycleSize = pInParams->IntRefCycleSize;
-        m_CodingOption2.IntRefQPDelta = pInParams->IntRefQPDelta;
+        co2->IntRefType = pInParams->IntRefType;
+        co2->IntRefCycleSize = pInParams->IntRefCycleSize;
+        co2->IntRefQPDelta = pInParams->IntRefQPDelta;
 
         if (pInParams->nExtBRC != EXTBRC_DEFAULT && (pInParams->EncodeId == MFX_CODEC_HEVC || pInParams->EncodeId == MFX_CODEC_AVC))
         {
-            m_CodingOption2.ExtBRC =(mfxU16)(pInParams->nExtBRC == EXTBRC_OFF ? MFX_CODINGOPTION_OFF : MFX_CODINGOPTION_ON);
+            co2->ExtBRC =(mfxU16)(pInParams->nExtBRC == EXTBRC_OFF ? MFX_CODINGOPTION_OFF : MFX_CODINGOPTION_ON);
         }
         else
         {
-            m_CodingOption2.ExtBRC = 0;
+            co2->ExtBRC = 0;
```

Comment:
let's use named const instead. This one is ```MFX_CODINGOPTION_UNKNOWN```

Reply:
ok